src/main/java/uk/gov/digital/ho/hocs/casework/api/dto/GetStageResponse.java

```diff
@@ -90,12 +91,14 @@ public class GetStageResponse {
     private String nextCaseStageUUID;
 
     public static GetStageResponse from(StageWithCaseData stage) {
+        var tags =
```

Comment:
This has 0(n) complexity on the return for cases that have a tag - but none for all other cases.

This is an improvement other the case data lookup that happened in `decorateTags` before.